resources/js/Pages/Adminland/Expense/Partials/Categories.vue

```diff
@@ -145,6 +146,16 @@ export default {
     };
   },
 
+  watch: {
+    categories(value) {
```

Comment:
why do you do this? i don’t see the need

Reply:
When you use the template in a page, you'll bind the value: `<categories :categories="valueFromPage" />`.
In the Categories component, the value is copied in `localCategories` to be used and changed. But if the `valueFromPage' change inside the Page after everything is mounted, you will have to update the `localCategories`. The `watch` function do that.